Task1/Patryk/main.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+#include <cstdio>
+
+using namespace std;
+
+bool isPrime(int value)
+{
+    bool isPrimeInd = true;
+    int i = 2;
+    do
```

Comment:
can you try to change 'do-while' loop into 'for' loop? 'for' might be less readable at first, but it will have advantage of keeping 'i' available only in scope where it's needed